test/validations.test.js

```diff
@@ -614,7 +614,52 @@ describe('validations', function() {
   });
 
   describe('exclusion', function() {
-    it('should validate exclusion');
+    it('fails when excluded value is used for property', function(done) {
+      User.validatesExclusionOf('name', {in: ['bob']});
+      User.create({name: 'bob'}, function(err, user) {
+        should.exist(err);
```

Comment:
We can remove this as we check on the next line. If you want to be more explicit, you can check for instanceOf error.